crates/libcontainer/src/syscall/linux.rs

```diff
@@ -743,6 +743,15 @@ impl Syscall for LinuxSyscall {
     fn get_egid(&self) -> Gid {
         nix::unistd::getegid()
     }
+
+    fn personality(&self, domain: u64) -> Result<()> {
+        use libc::personality;
+
+        match unsafe { personality(domain as libc::c_ulong) } {
```

Comment:
I think it's fine to use `libc::personality(domain)` just like in the other parts.